app/services/twilio_service.rb

```diff
@@ -25,7 +25,7 @@ def place_call(params = {})
 
   def send_sms(params = {})
     sanitize_errors do
-      params = params.reverse_merge(from: from_number)
+      params = params.reverse_merge(messaging_service_sid: Figaro.env.twilio_messaging_service_sid)
```

Comment:
What about `place_call`? Does Copilot only work for SMS?

Reply:
Yes, copilot only works for SMS

Reply:
Ah, that's a bummer. Would have been nice not to have to maintain that huge list of numbers in our application.yml

Reply:
Or does carrier filtering not apply to Voice calls?

Reply:
Yeah, but this does set us up to move to Verify if we want. That supports voice calls.

Also, my understanding is that carrier filtering is nowhere near as aggressive for voice calls as it is for SMS. Also shouldn't be as much of a concern since our volume of phone calls is much lower than for SMS.